### ISSUE TITLE
Better looking slashes and crosses in guitar tablature

### ISSUE THREAD

**Issue by ShibuyaCyana** [musescore/MuseScore#19084]
### Your idea

As in title

### Problem to be solved

These are slashes in Guitar Pro:
![image](https://github.com/musescore/MuseScore/assets/39703247/9fd236ef-fb77-445d-885e-9c69a11802d6)
And in MuseScore4:
![image](https://github.com/musescore/MuseScore/assets/39703247/c791a93a-9068-4eeb-a0dd-0270541dd29a)

The steps to make slashes and crosses in Guitar Pro is: 
1. select a note or a chord, click _slash_ in palette, then it becomes a big slash;
2. select a slash, click _dead note_ in palette, then it becomes a big cross;
3. if clicking _dead note_ when selecting note or chord instead of big slash, they will becomes small crosses instead.

Step 1 used to have bug in MU3 but fixed in MU4, tho the slashes don't look well in tablature, either version;
Step 2 is not achievable in MU3 or MU4. Trying to create a big cross in MU results in the picture above;
Step 3 is implemented in both MU3 and MU4, but the crosses looks weird in tablature. see also #11602 and #16500.

**Comment by rgreen5** [comment]
This should be a priority fix. Slash noteheads in tablature are unusable; without them you cannot write rhythm parts. This how they should look: [All Along the Watchtower (www.guitar.ch)](https://www.guitar.ch/tabs-pdf/tabs.php?pdf=JimiHendrix/AllAlongTheWatchtower).

**Comment by MarcSabatella** [comment]
FWIW, as a workaround it's easy enough to add slashes or large X noteheads from the Symbols palette to the notes on tablature.  You can also attach stems to these symbols and add the combined symbol to a palette for easy reuse.

But of course, a native solution would be nice too - ideally as part of a more generalized improved slash notation facility.

**Comment by ShibuyaCyana** [comment]
well, at least I can't find a way to make tablature numbers look like this; they do become small x and /, but not big slashes or crosses as shown in guitar pro screenshot. If there's a way please tell me.
And what is a combined symbol? sounds like a useful function, tho I did a google search and find nothing.

**Comment by MarcSabatella** [comment]
As I said, the workaround is to use the Symbols palette - you can reproduce that GP look pretty easily.  If you need help with that, please ask on the Support forum on musescore.org, so this issue tracker can be focused on the feature request itself.

**Comment by rgreen5** [comment]
> FWIW, as a workaround it's easy enough to add slashes or large X noteheads from the Symbols palette to the notes on tablature.
> 

AFAICS, it can't be done. If you hide all the underlying slashes, you hide the overlying slash + stem symbols as well.

**Comment by MarcSabatella** [comment]
That's true only if you hide the whole range.  Try hiding just note the notes.  Easiest to hide them as a range, before attaching the symbols.  Or hide everything and then go back and unhide the symbols.  It definitely works, although as noted, it's just a workaround.

**Comment by ShibuyaCyana** [comment]
So this is the step by step workaround, if someone really wanna do it for now:
https://musescore.org/en/node/357242

and a gif to make this comment more eye-catching:
![slashes](https://github.com/musescore/MuseScore/assets/39703247/f85d1911-14d1-4400-9b40-53576875be73)

@MarcSabatella @rgreen5 let's hide our derailed comments, continuing in musescore.org thread.
